frontend/app/assets/stylesheets/spree/frontend/views/spree/shared/main_nav_bar.scss

```diff
@@ -65,7 +65,7 @@
       border: none;
       z-index: 1000;
       display: none;
-      background-color: theme-color('light-background');
+      background-color: $meganav-background-color;
```

Comment:
@aplegatt let's make it more flexible and change it to `background`, not `background-color`